talk/app/talk-ui.hoon

```diff
@@ -1,5 +1,6 @@
 /-  u=ui, g=groups, c=chat
 /+  default-agent, dbug, verb, vita-client
+/%  init  %ui-init
```

Comment:
should this be `/$  init  %ui-init  %json`?

Reply:
yeah you're right

```suggestion
/$  init  %ui-init  %json
```